lib/helper/firefoxProfileCreator.py

```diff
@@ -72,7 +72,7 @@ def _install_profile_extensions(self, extensions_settings={}):
         return
 
     def remove_firefox_profile(self):
-        if os.path.isdir(self._firefox_profile_path):
+        if self._firefox_profile_path and os.path.isdir(self._firefox_profile_path):
```

Comment:
ok, this would make check faster, but you might want to comment on this for the reason.